docs/extensions/guides/renderer-extension.md

```diff
@@ -332,89 +332,7 @@ Global pages can be made available in the following ways:
 * To add global pages as an interactive element in the blue status bar along the bottom of the Lens UI, see [`statusBarItems`](#statusbaritems).
 * To add global pages to the left side menu, see [`globalPageMenus`](#globalpagemenus).
 
-### `globalPageMenus`
```

Comment:
In fixing https://github.com/lensapp/lens/issues/3065 I will be removing the `globalPageMenus` from `Renderer.LensExtension` and making on `PageMenuRegistration` a thing (instead of also having `ClusterPageMenuRegistration`.

This is just FYI, since I'll update the docs on my side.